modules/core/src/core-layers/text-layer/text-layer.js

```diff
@@ -37,10 +38,13 @@ const ALIGNMENT_BASELINE = {
 const DEFAULT_FONT_FAMILY = 'Monaco, monospace';
 const DEFAULT_COLOR = [0, 0, 0, 255];
 
+const MISSING_CHAR_WIDTH = 32;
```

Comment:
change to DEFAULT_CHAR_WIDTH and set as the fallback for getSize?

Reply:
I do think we need to redesign how size is handled. Maybe in a follow up PR.